packages/spec-api/src/types.ts

```diff
@@ -0,0 +1,83 @@
+import type { Request } from "express";
+import { MockRequest } from "./mock-request.js";
+import "multer";
```

Comment:
feels like that should be here

Reply:
@timotheeguerin I am not sure of the comment. I believe you want to remove the `import "multer"` statement. But, it is used down in the "Express.Multer" format and will cause build error if removed.

Reply:
That resolved the issue. Thanks